Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/ShaderStageFunction.h

```diff
@@ -18,16 +18,14 @@ namespace AZ
 
     namespace RHI
     {
-        /**
-         * Contains byte code associated with a specific entry point function of a shader stage. This
-         * data is provided to the PipelineStateDescriptor when building a PSO. Certain platforms may
-         * utilize function constants to specialize the same central byte code store. Thus, a
-         * ShaderStageFunction instance is a child of a ShaderStageLibrary container.
-         *
-         * Each platform specializes this data structure with platform-specific data necessary to compile
-         * an entry point of a shader stage on a PSO. The platform-independent runtime does not need to care
-         * about specifics, the function is merely an opaque data stream passed to the pipeline state descriptor.
-         */
+        //! Contains byte code associated with a specific entry point function of a shader stage. This
```

Comment:
This can be shifted over as well by using the AZ::RHI namespace as well.

The previous AZ namespace can be closed before opening up a new one.

```c++
namespace AZ
{
    class ReflectContext;
}

namespace AZ::RHI
{
    // ...
}
```